enterprise/server/util/redis/redis.go

```diff
@@ -0,0 +1,35 @@
+package redis
+
+import (
+	"log"
+	"strings"
+
+	redis "github.com/go-redis/redis/v8"
+)
+
+func isRedisURI(redisTarget string) bool {
+	return strings.HasPrefix(redisTarget, "redis://") ||
+		strings.HasPrefix(redisTarget, "rediss://") ||
```

Comment:
I can only ever read "rediss" in my head as if it's spoken by a snake 🐍

Reply:
🐍